.github/workflows/broker.yml

```diff
@@ -98,20 +98,36 @@ jobs:
     if: github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/')
     steps:
       - uses: actions/checkout@v2.3.4
+      - name: Setup qemu
+        uses: docker/setup-qemu-action@v1.2.0
       - name: Cache Docker layers
-        uses: actions/cache@v2.1.4
+        uses: actions/cache@v2.1.6
         with:
           path: /tmp/.buildx-cache
           key: ${{ runner.os }}-buildx-${{ github.sha }}
           restore-keys: |
             ${{ runner.os }}-buildx-
       - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1.1.1
+        id: buildx
+        uses: docker/setup-buildx-action@v1.5.1
         with:
           driver-opts: |
             network=host
+      - name: available platforms
+        run: echo ${{ steps.buildx.outputs.platforms }}
       - name: Build
-        uses: docker/build-push-action@v2.3.0
+        uses: docker/build-push-action@v2.7.0
+        with:
+          context: .
+          file: ./Dockerfile
+          platforms: linux/amd64,linux/arm64
+          push: false
+          load: false
```

Comment:
Set `load: true` and remove the step below:
`- name: Build (is cached!) and load x86 image`

Reply:
I tried this before, it doesn't work because you can't load multiple outputs of the step, it is a known issue and this is the workaround: https://github.com/docker/buildx/issues/59